### PR TITLE
Support for Channel Logging + Maintenance

### PR DESCRIPTION
**Pull request by MPW1412**
- Logs and shows Channel in 2,4 Ghz spectrum, 5 Ghz will thought IllegalArgumentException
- Updated .gitignore so that files, that are created during build process will be ignored
- Updated search pattern so support Freifunk written with small or capital letter

### CODE REVIEW

**Comment by tjanson** [comment]
Can you clarify why 5 GHz will throw an Ex.?

**Comment by MPW1412** [comment]
Yeah sorry, I used the wrong word, throw an IllegalArgumentException, not "thought".

It's in the new method I created, if the inputted frequency does not match the 2,4 spectrum, or if the function is called with any other int as parameter, an IllegalArgumentException will be thrown. 

It's just for documentation and will not happen during standard usecase. I like to have some kind of feature documentation via the change log in git.

File: app/src/main/java/com/tomjanson/wifilocationlogger/WifiBroadcastReceiver.java [hunk 1]
```diff
@@ -102,4 +103,14 @@ private Pattern makeFilter() {
         }
     }
 
+    public static int convertFrequencyToChannel(int freq) {
+        if (freq >= 2412 && freq <= 2484) {
+            return (freq - 2412) / 5 + 1;
+        } else if (freq >= 5170 && freq <= 5825) {
```
Comment (tjanson) [inline app/src/main/java/com/tomjanson/wifilocationlogger/WifiBroadcastReceiver.java:109]: This correctly converts 5 GHs networks, correct?

**Comment by MPW1412** [comment]
Ah shit, never mind. Product of copy and paste. Sorry for confusing. Just change the git commit comment during merging process.

**Pull request merged**